website/content/docs/secrets/databases/postgresql.mdx

```diff
@@ -84,6 +84,51 @@ the proper permission, it can generate credentials.
     username           v-vaultuse-my-role-x
     ```
 
+## Client x509 certificate authentication
+
+This plugin supports using PostgreSQl's [x509 Client-side Certificate Authentication](https://www.postgresql.org/docs/16/libpq-ssl.html#LIBPQ-SSL-CLIENTCERT).
+
+To use this authentication mechanism, configure the plugin to consume the
+PEM-encoded TLS data inline from a file on disk by prefixing with the "@"
+symbol. This is useful in environments where you do not have direct access to
+the machine that is hosting the Vault server. For example:
+
+```shell-session
+$ vault write database/config/my-postgresql-database \
+    plugin_name="postgresql-database-plugin" \
+    allowed_roles="my-role" \
+    connection_url="postgresql://{{username}}:{{password}}@localhost:5432/database-name?sslmode=verify-full" \
+    username="vaultuser" \
+    private_key=@/path/to/client.key \
+    tls_certificate=@/path/to/client.pem \
+    tls_ca=@/path/to/client.ca
+```
+
+Note: `private_key`, `tls_certificate`, and `tls_ca` map to [`sslkey`](sslkey_docs),
+[`sslcert`](sslcert_docs), and [`sslrootcert`](sslrootcert_docs) configuration
+options from PostgreSQL with the exception that the Vault parameters are the
+contents of those files, not filenames.
```

Comment:
LGTM though Vercel preview gives "Page not found" on these links (i.e. sslcert_docs)

Reply:
Thanks, fixed in https://github.com/hashicorp/vault/pull/28302/commits/164513963360068e8eb3ac1084ae3cdaac54abef